docs/README.md

```diff
@@ -1,12 +1,27 @@
-# [VC HTTP API](https://transmute-industries.github.io/vc-http-api/)
+# [VC HTTP API](https://github.com/w3c-ccg/vc-http-api/) Goals
 
-The following is a set of standard http api definitions for contructing and verifying objects which conform to the [VC Data Model](https://www.w3.org/TR/vc-data-model/).
+The W3C CCG VC HTTP API is a RESTful API definition conforming with the [OpenAPI 3.0 Specification](https://swagger.io/specification/) for constructing and verifying objects which conform to the [Verifiable Credential Data Model](https://www.w3.org/TR/vc-data-model/) specification.  This API provides a standard set of interfaces by which interoperability may be tested and verified by various parties who leverage Verifiable Credentials (VCs).
 
-This API is not meant to be exposed directly over http without authorization.
+As some implementations may not support all endpoints defined by this specification, the APIs also provide a clean measure by which to identify which methods are or are not implemented when comparing solutions that provide VC support across vendors.
 
-Implementations may not support all endpoints defined by this specification.
+As part of this API definition, test procedures and specifications are also provided to allow for repeatable and automated interoperability testing between solutions that interact with VCs.
 
-### Holder APIs are optional, many implementations will not need them.
+## Style, Versioning, and Contribution
+This API is versioned in coformance with the [Semantic Versioning 2.0 spec](https://semver.org/) to prevent breaking changes between major versions, and to allow for reliable testing and integration of implementations of this API within enterprise environments.
```

Comment:
```suggestion
This API is versioned in conformance with the [Semantic Versioning 2.0 specification](https://semver.org/) to prevent breaking changes between minor versions, and to allow for reliable testing and integration of implementations of this API within enterprise environments.
```